R/model.R

```diff
@@ -903,11 +979,30 @@ select_naomi_data <- function(
 
   naomi_mf$data_options <- data_options
 
+  # Combine and add full tagged data
+  survey_full_mf <- rbind(survey_prev_tagged$raw_input,
+                          survey_artcov_tagged$raw_input,
+                          survey_recent_tagged$raw_input,
+                          survey_vls_tagged$raw_input)
+
+  artnum_full_mf <- rbind(artnum_t1_tagged$raw_input,
+                          artnum_t2_tagged$raw_input) %>%
+    dplyr::distinct()
+
+  model_inputs <- list(survey_full_mf = survey_full_mf,
+                       artnum_full_mf = artnum_full_mf,
+                       anc_full_mf = anc_tagged$raw_input)
+
+  naomi_mf$model_inputs <- model_inputs
```

Comment:
Calling this `model_inputs` seems counterintuitive to me because it includes _all_ data vs. not only the data that are model inputs.  Can we call it something else?  E.g. `full_data`?  (I don't have any great ideas; just `model_inputs` seemed counterintuitive.)

Reply:
Have changed name to `full_data`